Cargo.toml

```
@@ -73,9 +73,11 @@ volume = ["libpulse-binding"]
 
 workspaces = ["futures-lite"]
 "workspaces+all" = ["workspaces", "workspaces+sway", "workspaces+hyprland"]
-"workspaces+sway" = ["workspaces", "swayipc-async"]
+"workspaces+sway" = ["workspaces", "sway"]
```

I'd actually rather this *was* duplicated, as it keeps it explicit & if the sway module deps change in the future it avoids bringing in to the workspace.

```suggestion
"workspaces+sway" = ["workspaces", "swayipc-async"]
````

---

Tried to do this, but `create_workspace_client` uses `sway::Client`, which is also is used by the `sway-mode` module. I can implement this change by carefully spreading `cfg(any(feature = "workspaces+sway", feature = "sway"))` in a couple of places, but I not sure if that is a win.

---

Actually, currently `sway::Client` is under `clients::compositor`, which is gated by `workspaces`. I first need to move the `sway::Client` out of there, it does not need to be gated by `workspaces`.